Sources/PackageRegistry/RegistryClient.swift

```diff
@@ -1095,8 +1095,15 @@ extension HTTPClientResponse {
         }
     }
 
-    fileprivate func validateAPIVersion(_ expectedVersion: RegistryClient.APIVersion = .v1) throws {
-        guard self.apiVersion == expectedVersion else {
+    fileprivate func validateAPIVersion(_ expectedVersion: RegistryClient.APIVersion = .v1, isOptional: Bool = false) throws {
+        let apiVersion = self.apiVersion
```

Comment:
nit: redundant?

Reply:
We use this so we don't need to call `self.apiVersion` twice.